viewShip.php

```diff
@@ -0,0 +1,52 @@
+<?php
+require __DIR__.'/bootstrap.php';
+// page to display a single ship data with the options to:
+// ADD, EDIT, DELETE
+use Service\Container;
+
+
+
+$id = $_GET['id'];
+
+// Need shiploader obj for ship
+$container = new Container($configuration);
+$shipLoader = $container->getShipLoader();
+$ship = $shipLoader->findOneById($id); // values may be getting changed here
```

Comment:
There should be some error checking around this:

1. Make sure the 'id' parameter is in the GET request
2. Show a message a ship is not found (in invalid id is passed in)